### ISSUE TITLE
Zigbee2MQTT keeps crashing since updating to 1.25.0 with error (0xc7: NWK_TABLE_FULL)' (expected '(0x00: SUCCESS)')

### ISSUE THREAD

**Issue by Devon1984** [Koenkk/zigbee2mqtt#12059]
### What happened?

Since updating to 1.25.0 my Z2M container keeps crashing after a few hours with the following displayed in the logs.

Error: SREQ '--> ZDO - extRouteDisc - {"dstAddr":17327,"options":0,"radius":30}' failed with status '(0xc7: NWK_TABLE_FULL)' (expected '(0x00: SUCCESS)')
at Object.func (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:322:27)
at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

If i restart the container it works fine again for another few hours and then crashes.
This has never happened with any other version previously.

I have now rolled back to 1.24.0 and my Z2M has been stable since the downgrade.

Not sure if this makes a difference but im running this on my Unraid server version 6.9.2 with Linux kernal 5.10.28 and docker version 20.10.5

### What did you expect to happen?

after the update to 1.25.0 i expected it to be stable.

### How to reproduce it (minimal and precise)

Update to version 1.25.0

### Zigbee2MQTT version

1.25.0

### Adapter firmware version

20220219

### Adapter

Electrolama zig-a-zig-ah! CC2652R

**Comment by Devon1984** [comment]
Update....
Not sure if @Koenkk has made any changes in his Dev branch since 02-04-22 that could have fixed this issue but i decided to try the 1.25.0 dev branch again yesterday afternoon. I also enabled the Advanced Device availability function and set all my active device timeouts to 60 min and since then zigbee2mqtt hasnt crashed yet which is a good sign....

I have seen 5 of the NWK_TABLE_FULL events but nothing more so fingers crossed :)

Ill keep watching though and keep this updated.

**Comment by Devon1984** [comment]
Closing this issue since it hasn't crashed for the week.
Just in case anyone else has this same issue, this is what i did which seemed to have fixed it.

Bit of background....
I was initially on the 1.25.0 dev branch when the crashing events started happening. I then rolled back to 1.24.0 stable and all was fine again. Once 1.25.0 stable came along the crashing events started happening again. I changed back to 1.25.0 dev branch and the crashing events started happening again.... thats when i posted my issue on here. I then rolled back to 1.24.0 stable for 3 days and no crashing events happened at all.

After 3 days on the 1.24.0 stable version i decided to do 2 things,
1 - I went back to the 1.25.0 dev branch
2 - I enabled "Availability (Advanced)" in the zigbee2mqtt front end and changed the active devices timeout initially to 60 but after a few days dropped it down to 30.

Anyway, either the new updated 1.25.0 dev branch fixed the issue or it was me reducing the amount of pinging going on to all my devices (i have a lot of devices).
Also, network interference was definitely not an issue, my zigbee channel has always bee on 11 and my 2.4Ghz wifi is permanently on channels 11, 12 and 13.
